### ISSUE TITLE
Change the key binding for "Toggle math" from `Ctrl+M` to `Alt+=`

### ISSUE THREAD

**Issue by heartacker** [yzhang-gh/vscode-markdown#1007]
The original proposal was invalid because none of the key combinations can be triggered on a real-world keyboard.

## Candidates

* `Ctrl + M` (`⌘ + M`)

  * Pro:
    * The current approach. `M` is the abbreviation for "Math".

  * Con:
    * `Ctrl + M` overrides "Toggle Tab Key Moves Focus" (`editor.action.toggleTabFocusMode`) on Windows and Linux.
    * `⌘ + M` overrides "Minimize the front window to the Dock" on macOS.

* `Alt + =` (`⌥ + =`)

  * Pro:
    * `Alt + =` is from Office for Windows.

  * Con:
    * `Alt + =` overrides "Search Editor: Increase Context Lines" (`increaseSearchEditorContextLines`).

* `Ctrl + =` (`⌃ + =`)

  * Pro:
    * `⌃ + =` is from Office for macOS.

  * Con:
    * `Ctrl + =` overrides "View: Zoom In" (`workbench.action.zoomIn`) on Windows and Linux. (`⌘ + =` on macOS)

## References

* <https://code.visualstudio.com/docs/editor/accessibility>

**Comment by yzhang-gh** [comment]
> How about `Alt + =`? As used in Office.

Interesting. Sounds like a candidate choice

**Comment by Lemmingh** [comment]
`=` is "Equation". But our feature is named "Math" (`M`). I still think `M` is the most intuitive.

**Comment by Yue-Zhengyuan** [comment]
In fact, on macOS the current shortcut `command + M` conflicts with window minimization; I also support using the same shortcut as MS Office, but on macOS the shortcut is `control + =` instead of `alt(option) + =`

**Comment by Lemmingh** [comment]
> on macOS the current shortcut `command + M` conflicts with window minimization

<https://github.com/yzhang-gh/vscode-markdown/issues/832#issuecomment-705984794>?

**Comment by Yue-Zhengyuan** [comment]
@Lemmingh Indeed; VS Code will override the system shortcut when editing an md file with the extension on. On other files which does not need this extension, `command + M` will become window minimization again.
